hw1/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Jersey</title>
+</head>
+<body>
+    <div class="weather">
+        <div class="weather__top">
+            <div class="wt__item">
+                <img src="images/3.png">
+                <div class="wt__refresh"><img src="images/refresh.png"></div> 
+            </div>
+        </div>
+
+        <div class="weather__middle">
+          <div class="wm__item">
+            <div class="wm__degrees">79</div>
+            <div class="o">o</div>
+        </div>
+        <div class="wm__item_middle">
+            <div class="wm__weather">Cloudy Skies</div>
+            <div class="wm__location">Sicklerville, New Jersey </div>
+        </div>
+        <div class="wm__lastitem">
+            <div class="wm__month">MAY</div>
+            <div class="wm__date">21</div>
+        </div>
+    </div>
+
+    <div class="weather_bottom">
```

Comment:
It is important to use semantic tags. Try to replace divs with semantic tags were it will be appropriate.
For example, you can use `<header>`, `<footer>`, `<main>`.
 
https://www.freecodecamp.org/news/semantic-html5-elements/